core/tests/behavior/async_list.rs

```diff
@@ -558,6 +563,11 @@ pub async fn test_list_dir_with_recursive_no_trailing_slash(op: Operator) -> Res
 }
 
 pub async fn test_list_file_with_recursive(op: Operator) -> Result<()> {
+    // Dropbox does not support list file with recursive.
```

Comment:
It doesn't make sense. All services that implement list_dir should have list prefix.

Reply:
I'm not sure about this, I don't see it described in the [API documentation](https://www.dropbox.com/developers/documentation/http/documentation#files-list_folder), and in my practice Dropbox doesn't implement prefix matching.

Reply:
prefix matching is simulated at https://github.com/apache/opendal/blob/755de5829a127e3e1f9da334e69cf4436578f889/core/src/layers/complete.rs#L276-L322

I have a plan to refactor them later to make those behavior more clear.